iio/iio_ad9361/iio_ad9361.c

```diff
@@ -2296,10 +2296,6 @@ int32_t iio_ad9361_init(struct iio_ad9361_desc **desc,
 	iio_ad9361_inst->dev_descriptor.attributes = global_attributes;
 	iio_ad9361_inst->dev_descriptor.debug_attributes = NULL;
 	iio_ad9361_inst->dev_descriptor.buffer_attributes = NULL;
-	iio_ad9361_inst->dev_descriptor.transfer_dev_to_mem = NULL;
```

Comment:
It looks that you are doing here changes also to other projects that are unrelated to ad7134.